sign_oca/security/security.xml

```diff
@@ -14,9 +14,113 @@
         <field name="name">User: All Documents</field>
         <field name="category_id" ref="sign_oca_module_category" />
         <field name="implied_ids" eval="[(4, ref('sign_oca_group_user'))]" />
+    </record>
+    <record id="sign_oca_group_admin" model="res.groups">
+        <field name="name">Administrator</field>
+        <field name="category_id" ref="sign_oca_module_category" />
+        <field name="implied_ids" eval="[(4, ref('sign_oca_group_manager'))]" />
         <field
             name="users"
             eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]"
         />
     </record>
+    <!-- Record Rules -->
+    <record id="sign_oca_request_signer_rule_base_user_read" model="ir.rule">
+        <field name="name">Sign Request base users: read</field>
+        <field name="model_id" ref="model_sign_oca_request_signer" />
+        <field
+            name="domain_force"
+        >[('partner_id', 'child_of', [user.partner_id.commercial_partner_id.id])]</field>
+        <field name="groups" eval="[(4, ref('base.group_user'))]" />
+        <field name="perm_read" eval="1" />
+        <field name="perm_create" eval="0" />
+        <field name="perm_write" eval="0" />
+        <field name="perm_unlink" eval="0" />
+    </record>
+    <!-- User Record Rules -->
+    <record id="sign_oca_template_rule_user_read" model="ir.rule">
+        <field name="name">Sign Oca Template user: read</field>
```

Comment:
As a question/remark in general. Do we need all the fields to be OCA branded?
Does not add value to the end user.
It is very unlikely someone will install odoo sa and oca's sign module.
Adn dev's will look at xmlid anyway.

Reply:
Removed OCA from the name.